Dockerfile

```diff
@@ -6,8 +6,8 @@ ENV CYPRESS_VIDEO=false
 
 WORKDIR /e2e
 
-COPY ./package.json ./package.json
-RUN npm install
+COPY package.json yarn.lock ./
+RUN yarn
```

Comment:
Should we use `yarn install --frozen-lockfile`?

Reply:
Thank you! I added the flag